pkg/ottl/contexts/internal/errors.go

```diff
@@ -0,0 +1,22 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package internal // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/contexts/internal"
+
+import "fmt"
+
+const (
+	DefaultErrorMessage = "%v is not a valid path for the %v context - review %v to see all valid paths"
```

Comment:
```suggestion
	DefaultErrorMessage = "%q is not a valid OTTL keyword nor a valid path for the %v context - review %v to see all valid paths"
```
Referring back to the issue, I think we should make sure to call out that the parsing error could be due to a bad keyword, e.g. if `null` was used instead of `nil`.

Also I think it may help to quote the path just to make it a little clearer.